src/providers/twitch/TwitchAccount.cpp

```diff
@@ -285,8 +304,13 @@ void TwitchAccount::loadUserstateEmotes(std::weak_ptr<Channel> weakChannel)
                         continue;
                     }
 
+                    emoteSet->channelID = ivrEmoteSet.channelId;
                     emoteSet->channelName = ivrEmoteSet.login;
                     emoteSet->text = ivrEmoteSet.displayName;
+                    emoteSet->subscriber = !ivrEmoteSet.tier.isNull();
+
+                    bool haveSubscriberSetForChannel =
+                        subscriberChannelIDs.contains(ivrEmoteSet.channelId);
```

Comment:
Edge case: if user is subscribed to a channel that offers follower emotes but no sub emotes, `haveSubscriberSetForChannel` would be false, leading to `local = true`

that said, this edge case is rare, the PR is a significant improvement, and fixing this edge case is likely not worth the complexity/additional IVR requests